Added note on Docker passwords change (#6554)
wazuh/wazuh-documentation: source/deployment-options/docker/wazuh-container.rst

Wazuh indexer users
~~~~~~~~~~~~~~~~~~~

>  To change the password of the default ``admin`` and ``kibanaserver`` users, do the following. They can only be changed one at a time. 

Reviewer: ```suggestion
 To change the password of the default ``admin`` and ``kibanaserver`` users, do the following. You can only change one at a time. 
```

Author: Done in https://github.com/wazuh/wazuh-documentation/pull/6554/commits/ec962baf1eaa5d64ec7831158695470b4c101785.
